source/_integrations/home_connect.markdown

```diff
@@ -43,7 +43,10 @@ Note that it depends on the appliance and on API permissions which of the featur
 - OAuth Flow: Authorization Code Grant Flow
 - Redirect URI: `https://my.home-assistant.io/redirect/oauth`
 
-*Important*: after performing the steps above **log out** of your Home Connect Developer account. If you don't so this, the configuration steps below will fail during OAuth authentication with the message `“error”: “unauthorized_client”`.
+*Important*: 
+ - after performing the steps above **log out** of your Home Connect Developer account. If you don't so this, the configuration steps below will fail during OAuth authentication with the message `“error”: “unauthorized_client”`.
+ - The provided Home Connect User Account email address **must** be all lowercase otherwise it will result in authentication failures.
```

Comment:
This is not something that needs documentation, this is something that needs fixing. The integration can just make it lower case before sending it out, removing the frustration.

Reply:
Hi @frenck unfortunately this has to be done on the side of home connect. Because the oauth flow is used home assistant only works with a bearer token

Reply:
You are absolutely right. (darn it, would be nice to fix this).